public/stylesheets/main.css

```diff
@@ -1,32 +1,43 @@
 body {
-  font-family: "Helvetica Neue",Helvetica,Arial,sans-serif;
-  font-size: 14px;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
   line-height: 1.428571429;
   color: #333;
 }
 
 #logo {
   text-align: center;
+  float:left;
+  margin-top:1em;
 }
 
 #content {
-  padding: 1em;
+  /*padding: 1em;*/
```

Comment:
Don't be afraid to delete unwanted style declarations! If you don't need them, they're just noise. They'll always be around in the git source control repository if they need to be reinstated, or if you just need to refer to them in general.